kythe/go/platform/analysis/proxy/proxy.go

```diff
@@ -110,13 +122,20 @@ type response struct {
 	Args   interface{} `json:"args,omitempty"`
 }
 
-// A unit represents a compilation unit to be analyzed.
+// A unit represents a compilation unit (in JSON form) to be analyzed.
 type unit struct {
 	Unit            json.RawMessage `json:"unit"`
 	Revision        string          `json:"rev,omitempty"`
 	FileDataService string          `json:"fds,omitempty"`
 }
 
+// A unit represents a compilation unit (in base64-encoded wire format) to be analyzed.
+type unit_wire struct {
```

Comment:
Names in Go should be `mixedCase`, not `snake_case`, for multiple words. See: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps (or the internal analogue).

Reply:
Ah good catch!